docs/operation-guide.md

```diff
@@ -0,0 +1,129 @@
+
+```shell
+$ kubectl delete pvc -n ${namespace} -l app.kubernetes.io/instance=${releaseName},app.kubernetes.io/managed-by=tidb-operator
+$ kubectl get pv -l app.kubernetes.io/namespace=${namespace},app.kubernetes.io/managed-by=tidb-operator,app.kubernetes.io/instance=${releaseName} -o name | xargs -I {} kubectl patch {} -p '{"spec":{"persistentVolumeReclaimPolicy":"Delete"}}'
+```
+
+> **Note:** the above command will delete the data permanently. Think twice before executing them.
+
+## Monitor
+
+TiDB cluster is monitored with Prometheus and Grafana. When TiDB cluster is created, a Prometheus and Grafana pod will be created and configured to scrape and visualize metrics.
+
+By default the monitor data is not persistent, when the monitor pod is killed for some reason, the data will be lost. This can be avoided by specifying `monitor.persistent` to `true` which will create a persistent volume for storing monitor data.
+
+You can view the dashboard using `kubectl portforward`:
+
+```shell
+$ kubectl port-forward -n ${namespace} svc/${clusterName}-grafana 3000:3000 &>/tmp/portforward-grafana.log
+```
+
+Then open your browser at http://localhost:3000 The default username and password are both `admin`
+
+The Grafana service is exposed as `NodePort` by default, you can change it to `LoadBalancer` if the underlining Kubernetes has load balancer support. And then view the dashboard via load balancer endpoint.
+
+## Backup
+
+Currently, TiDB Operator supports two kinds of backup: full backup via [Mydumper](https://github.com/maxbube/mydumper) and incremental backup via binlog.
+
+### Full backup
+
+Full backup can be done periodically just like crontab job. Currently, full backup requires a PersistentVolume, the backup job will create a PVC to store backup data.
+
+To create a full backup job, modify charts/tidb-cluster/values.yaml `fullbackup` section.
+
+* `create` must be set to `true`
+* Set `storageClassName` to the PV storage class name used for backup data
+* `schedule` takes the [Cron](https://en.wikipedia.org/wiki/Cron) format
+* `user` and `password` must be set to the correct user which has the permission to read the database to be backuped.
+
+If TiDB cluster is running on GKE, the backup data can be uploaded to GCS bucket. A bucket name and base64 encoded service account credential that has bucket read/write access must be provided. The comments in charts/tidb-cluster/values.yaml is self-explanatory for GCP backup.
+
+### Incremental backup
+
+To enable incremental backup, set `binlog.pump.create` and `binlog.drainer.create` to `true`. By default the incremental backup data is stored in protobuffer format in a PV. You can change `binlog.drainer.destDBType` from `pb` to `mysql` or `kafka` and configure the corresponding downstream.
+
+## Restore
+
+Currently, tidb-operator only supports restoring from full backup in GCS bucket. The `restore` section in charts/tidb-cluster/values.yaml should have enough comments as document.
```

Comment:
I don't think the restore workflow makes sense. The helm chart is for persistent objects, but restore is a one-off operation. There could be multiple restores of different databases or different restores at different times, and I don't know how `helm upgrade` will handle it. Can we separate out the restore operation from the helm chart?

Reply:
The restore section itself documents what fields can be changed, but not any of the workflow concerns.